### ISSUE TITLE
Vocabulaire accessible : changer le terme **CONTRAT**

### ISSUE THREAD

**Issue by jeanlucchristin** [jeanlucchristin/La_demarche_gouvernance_cellulaire#99]
Bonjour, 

Des termes rendent la constitution difficile d'accès pour des public traditionnels, déroutés par de nouveaux mots qui ont tendance à les rejeter. 
J'ai envie de rendre l'accès à la constitution hyper facile pour le plus grand nombre. 
Par conséquent, je propose de modifier certains termes. 

La proposition du jour : remplacer **CONTRAT** par **POUVOIR**. 
![image](https://user-images.githubusercontent.com/38285180/207384437-41c24e61-f442-4f72-a020-5f62624fb7a3.png)

Le mot Contrat est édulcoré. Il parle très bien de ce qui peut être fait (clarification des rôles, des processus de décision, etc) mais pas de ce qui se joue (répartition du pouvoir). 
Or gouvernance cellulaire vise la transparence, la légitimité et l'exercice du pouvoir. 

Je crois que le mot pouvoir véhiculera mieux la notion qui est au travail que le mot contrat, avec lequel tout le monde est d'accord mais sans prendre en compte les enjeux, même pas intuitivement. 

**Des avis des réactions sur cette proposition ?**

**Comment by DavidDrayer** [comment]
Je rejoins Christophe sur le danger à changer trop souvent les termes... déjà que dans ma pratique je dois jongler entre 3 langages différents, si en plus ces différents langages changent régulièrement, ça devient un vrai casse-tête pédagogique.

Je suis capable aujourd'hui de défendre le terme contrat: la structure, c'est un contrat passé ensemble sur qui fait quoi, validé collectivement. Un projet, c'est un contrat interne passé par deux personnes, où l'une s'engage à faire quelque chose pour l'autre. 
A ce jour, je n'ai pas d'explication qui me vienne sur "pouvoir", donc je veux bien un détail de l'approche pédagogique sous-jacente.

Et pour les espaces, nous avons le même défi à l'instant Z avec nos 5 espaces, que nous devons décliner à chaque fois avec leur définitions:
Sens : La raison d'être et le cadre éthique de l'orga
Rêve : Les objectifs et les priorités à moyen terme
Plan : La structure et les règles
Action : Les opérations et la circulation de l'information
Célébration : L'ajustement agile et les relations

**Comment by DavidDrayer** [comment]
Petit ajout, nous venons de valider le passage à 6 espaces, qui curieusement nous rapproche de gouvernance cellulaire:
Nous avons l'espace **Sens** (le tout), qui s'attache à prendre soin du monde
L'espace **Contrat** (le nous), que nous subdivisons en PDCA (Rêve, Plan, Action, Célébration, que l'on pourrait nommer également Stratégie, Planification, Coordination et Amélioration continue), qui s'attache à prendre soin de l'activité
Et l'espace **Relation** (le je) qui s'attache à prendre soin des individus.

**Comment by jeanlucchristin** [comment]
Les avis vont tous dans le même sens et le besoin de changer de terme ne me parait pas fondamental aujourd'hui.

Le texte gouvernance cellulaire n'est pas forcément le bon endroit pour conscientiser l'importance de la répartition du pouvoir

Je vais clore cette issue, elle réapparaitra éventuellement dans le futur si le besoin revient.

**Comment by jeanlucchristin** [comment]
Bonjour, 

Le sujet du terme "contrat" réapparait, je réouvre cette issue. 

Proposition du jour : ESPACE ORGANISATION

cet espace contient le rôles (qui fait quoi ?), les processus, en particulier de décision, les politiques, etc. 
On l'appelait "espace contrat", peut-on l'appeler "espace organisation ?

L'objectif est d'avoir le terme le plus intuitif pour les utilisateurs finaux.

**Comment by Nicobe73** [comment]
J'aime bien le mot "Organisation" par rapport à "Contrat". C'est une façon aussi de montrer que la gouvernance est bien plus que l'organisation.

**Comment by christophe-herault** [comment]
Je suis d'accord que le terme "espace contrat" n'est pas très intuitif et accessible à nos clients. Je trouve intéressant de trouver un terme plus usuel. Je suis réticent à le remplacer par "organisation". Selon moi, l'organisation est une sous-partie de l'espace contrat. Je vois dans l'espace contrat 4 dimensions : 
- l'organisation (définir comment on travail ensemble), 
- les priorités stratégiques (se mettre d'accord comment nous allons réaliser notre raison d'être), 
- l'opérationnel (travailler ensemble), 
- et la rétrospective (apprendre ensemble de nos expériences pour améliorer notre fonctionnement sur les 3 points précédents).

Aussi, je propose de rechercher un terme qui englobe toutes ces notions. Je pense que ça correspond à ce qu'Eric Berne décrit comme "l'activité" dans la TOB. Je serais assez confortable avec ce terme. 3 Espaces : **Activité**, Lien et Sens. 
Je serais aussi à l'aise avec les termes "**Coopération**" ou "**Action**".

**Comment by jeanlucchristin** [comment]
Tant qu'à envisager "activité", autant aller jusqu'à "action" ? 

Espace Sens
Espace Lien 
Espace Action

**Comment by christophe-herault** [comment]
Cette proposition me convient bien :
Espace Sens
Espace Lien
Espace Action


Christophe




________________________________
De : Jeanluc Christin ***@***.***>
Envoyé : jeudi 4 avril 2024 22:05
À : jeanlucchristin/La_demarche_gouvernance_cellulaire ***@***.***>
Cc : Christophe Herault ***@***.***>; Comment ***@***.***>
Objet : Re: [jeanlucchristin/La_demarche_gouvernance_cellulaire] Vocabulaire accessible : changer le terme **CONTRAT** (Issue #99)


Tant qu'à envisager "activité", autant aller jusqu'à "action" ?

Espace Sens
Espace Lien
Espace Action

**Comment by GeorgesComby** [comment]
Joyeux de lire cette proposition



Bien à vous tous

Georges Comby
06 86 16 77 86
Pas 💯% parfait, juste 💯% vivant, énergique & dynamique


-------- Message original --------
De : christophe-herault ***@***.***>
Date : lun. 8 avr. 2024 à 11:41
À : jeanlucchristin/La_demarche_gouvernance_cellulaire ***@***.***>
Cc : Subscribed ***@***.***>
Objet : Re: [jeanlucchristin/La_demarche_gouvernance_cellulaire] Vocabulaire accessible : changer le terme **CONTRAT** (Issue #99)

Cette proposition me convient bien :
Espace Sens
Espace Lien
Espace Action


Christophe




________________________________
De : Jeanluc Christin ***@***.***>
Envoyé : jeudi 4 avril 2024 22:05
À : jeanlucchristin/La_demarche_gouvernance_cellulaire ***@***.***>
Cc : Christophe Herault ***@***.***>; Comment ***@***.***>
Objet : Re: [jeanlucchristin/La_demarche_gouvernance_cellulaire] Vocabulaire accessible : changer le terme **CONTRAT** (Issue #99)


Tant qu'à envisager "activité", autant aller jusqu'à "action" ?

Espace Sens
Espace Lien
Espace Action

**Comment by jeanlucchristin** [comment]
Hello

J'ai testé activité et action et il y a trop d'incertitude : ces termes entraînent les gens vers les réunion métier, le "faire". Plus que la structuration, le qui fait quoi etc. 

A 15 jours de la release, ça va trop vite pour valider le caractère intuitif de "action" ou "activité".

"Organisation" apporte manifestement une meilleure compréhension que "contrat", je vais retenir cette proposition de @christophe-herault.

**Comment by DavidDrayer** [comment]
Et bien ça m'arrange, c'est ce que j'allais écrire:
Avec la démarche Z, nous tentons de faire des pas en direction de la Gouvernance Cellulaire pour faciliter l'imbrication (entre autre avec la création de l'espace relation), mais ce terme "action" était perturbant pour nous, puisqu'il s'agissais uniquement d'une dimension de nos 4 espaces "projet" inspirés par le rêve du dragon.

**Comment by jeanlucchristin** [comment]
Issue fermée car la modification a été entérinée dans la version de l'an passé
